### PR TITLE
Update docs

### LINKED ISSUE #82: IXSI mapping of station_information.capacity
Please document in https://github.com/mobidata-bw/x2gbfs/blob/main/docs/mappings/ixsi_gbfs_2.3_mapping.md#station_informationjson the mapping for capacity.

### PR DESCRIPTION
**Pull request by hbruch**
This PR 

* adds/fixes capacity mapping documentation for MOQO and Cantamen
* updates README to include currently supported providers

fixes #82

### CODE REVIEW

**Review by ThorstenFroehlinghaus — COMMENTED** [review]
Carsharing Südtirol should not be mentioned.

File: README.md [hunk 1]
```diff
@@ -33,7 +33,14 @@ be provided via config/<provider>.json and needs to be updated when that informa
...
 ## Available providers
 
-Currently, a couple of providers are supported: deer, which uses Fleetster as backend provider, Lastenvelo Freiburg, VOI Karlsruhe via backend provider Raumobil, and my-e-car and Stadtmobil Südbaden via backend provider IXSI.
+Currently, a couple of providers are supported:
+
+* deer, which uses Fleetster as backend provider,
+* Lastenvelo Freiburg
+* VOI Karlsruhe via backend provider Raumobil
+* Stadtwerk Tauberfranken via backend provider MOQO
+* Carsharing Südtiro via NOI's OpenDataHub
```
Comment (derhuerst) [inline README.md:42]:
```suggestion
* Carsharing Südtirol via NOI's OpenDataHub
```

[the remark this answers is not in the document]
Comment (derhuerst) [reply README.md:42]: We're documenting supported features of `x2gbfs` here, even if they're not being used in [`ipl-orchestration`](https://github.com/mobidata-bw/ipl-orchestration).

**Review by derhuerst — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (hbruch) [reply README.md]: Added in [8bf902c](https://github.com/mobidata-bw/x2gbfs/pull/86/commits/8bf902c3d1724cbc88048818fba3386925fe71d6)

**Pull request merged**